src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -320,6 +322,28 @@ class UnifiedShareModal extends React.Component<Props, State> {
         });
     };
 
+    onPillCreate = (type: string) => (pills: Array<SelectOptionProp | Contact>) => {
+        // If this is a dropdown select event, we ignore it
+        if (pills.length === 1 && pills[0].id) {
```

Comment:
I would not make the assumption that this function only ever gets 1 pill. Instead, I would just filter `pills` 
```
const selectOptionPills = pills.filter(pill => !pill.id);
if (selectOptionPills.length > 0) {
    return;
}

const emails = pills.map(pill => pill.value);
```

Reply:
done